doc/setup.md

```diff
@@ -49,13 +49,32 @@ If you are using Windows Subsystem for Linux (WSL) be sure you clone the reposit
 
 These instructions assume that you have [Homebrew](http://brew.sh/) installed.
 
-1. Install dependencies
+1. Install project dependencies
 
-	```
-	brew install supervisor nginx postgresql node \
-	     llvm libomp gsl rust
-	nvm install node
-	```
+Using Homebrew, install core dependencies:
+```
+brew install supervisor
+brew install nginx
+brew install postgresql
+brew install node
+brew install llvm
+brew install libomp
+brew install gsl
+brew install rust
+```
+Then, install Node.js with NVM:
+```
+nvm install node
+```
+Finally, install the scientific libraries:
```

Comment:
These aren't really scientific, just compression algorithms (like ZIP or GZIP). Here I would mention that they are needed to install the python dependency `tables` later on.